__tests__/app.test.js

```diff
@@ -55,31 +56,82 @@ describe('ERROR GET /api/topicd', () => {
 describe('GET /api/articles/:article_id', () => {
   test('sends GET to ID endpoint -> returns an object', () => {
     return request(app)
-    .get('/api/articles/1')
-    .expect(200)
-    .then((res) => {
-      const output = res.body.rows[0];
-      expect(output).toBeInstanceOf(Object);
-      expect(Object.keys(output).length).toBe(7);
+      .get('/api/articles/1')
+      .expect(200)
+      .then((res) => {
+        const output = res.body.rows[0];
+        expect(output).toBeInstanceOf(Object);
+        expect(Object.keys(output).length).toBe(7);
     });   
   });
 
   test('sends GET to ID endpoint -> checks each object key', () => {
     return request(app)
-    .get('/api/articles/1')
-    .expect(200)
-    .then((res) => {
-      const output = res.body.rows[0];
-      expect(output).toHaveProperty('author');
-      expect(output).toHaveProperty('title');
-      expect(output).toHaveProperty('article_id');
-      expect(output).toHaveProperty('body');
-      expect(output).toHaveProperty('topic');
-      expect(output).toHaveProperty('created_at');
-      expect(output).toHaveProperty('votes');
-    });   
+      .get('/api/articles/1')
+      .expect(200)
+      .then((res) => {
+        const output = res.body.rows[0];
+        expect(output).toHaveProperty('author');
+        expect(output).toHaveProperty('title');
+        expect(output).toHaveProperty('article_id');
```

Comment:
if we are checking for a specific article then we would want to be a bit more specific with the testing. 
what do we expect the article_id to be? etc.